src/backend/updater.ts

```diff
@@ -7,10 +7,10 @@ import { logError, LogPrefix, logInfo } from './logger/logger'
 import { isOnline } from './online_monitor'
 
 autoUpdater.autoDownload = true
-autoUpdater.autoInstallOnAppQuit = true
+autoUpdater.autoInstallOnAppQuit = false
```

Comment:
don't we need this true?

Reply:
Yeah, it changes if the user clicks NO just bellow.